Command Line Tools/Update Languages/WikipediaLanguageCommandLineUtilityAPI.swift

```diff
@@ -44,10 +44,19 @@ class WikipediaLanguageCommandLineUtilityAPI {
                     else {
                         return nil
                 }
-                return Wikipedia(languageCode: code, languageName: name, localName: localname)
+                
+                guard code == "no" else {
```

Comment:
Minor quibble, not a bug, feel free to ignore: I'd sorta expect to see the `guard` check if it's not `no` - and if it is `no` (and thus non-standard), let the guard statement handle that case.

Reply:
Good catch! This is fixed.